libsolidity/interface/FileReader.cpp

```diff
@@ -113,19 +113,23 @@ ReadCallback::Result FileReader::readFile(string const& _kind, string const& _so
 		vector<reference_wrapper<boost::filesystem::path>> prefixes = {m_basePath};
 		prefixes += (m_includePaths | ranges::to<vector<reference_wrapper<boost::filesystem::path>>>);
 
+		vector<boost::filesystem::path> searchedPaths;
 		for (auto const& prefix: prefixes)
 		{
 			boost::filesystem::path canonicalPath = normalizeCLIPathForVFS(prefix / strippedSourceUnitName, SymlinkResolution::Enabled);
-
+			searchedPaths.push_back(canonicalPath);
```

Comment:
This adds the whole path, including file name, not just the directory. The directory is just `prefix`. But if you want to print the prefixes, you don't even need `searchedPaths`. You can just use `prefixes`.

Reply:
`searchedPaths` also has `normalizeCLIPathForVFS` - can we just ignore it (should we)?

Reply:
Right. Depends on whether we prefer to print paths as the user specified them on the CLI or the normalized versions we actually use. I guess the latter might be more helpful in case the path had `.`, `..`, `~` in it or had symlinks and the user expected them to be resolved (they're not).

So yeah, in that case `searchedPaths` is fine but we could also just add a lambda that converts them when the message is generated instead of storing them.

In any case the source unit names should not be included.